### ISSUE TITLE
Windows audio in Qubes doesn't work

### ISSUE THREAD

**Issue by FireFox990** [duncanthrax/scream#144]
Hello, I have installed Windows 10 in separate VM on Qubes OS and It all works well but audio doesn't
I have installed scream on Win10 and it showed up that audio drivers are working ,but I can't hear anything 
how do I fix this issue?

**Comment by duncanthrax** [comment]
Did you start the receiver on the host system?

**Comment by FireFox990** [comment]
no, I just installed scream on W10 virtual machine and thats it, please explain me what do you mean by receiver and how do I go about it.
